canonical-websites/www.canonical.com
# Upgrade homepage to v1

Note: All other template pages will be touched and proposed section by section.

## Done

Upgrade homepage to Vanilla v1 styling

## QA

- Pull code
- Run `./run serve --watch`
- View homepage at http://0.0.0.0:8002/
- Check template code

## Issue / Card

Fixes https://www.github.com/ubuntudesign/vanilla-squad/issues/26

## Demo

http://www.canonical.com-homepage-polish.demo.haus/
